packages/db/lib/gen-migration.mjs

```diff
@@ -0,0 +1,66 @@
+import { readdir, stat, writeFile } from 'fs/promises'
+import path from 'path'
+import loadConfig from './load-config.mjs'
+
+async function generateMigration (_args) {
+  const { configManager } = await loadConfig({}, _args)
+  const migrationsDir = configManager.current.migrations.dir
```

Comment:
You read migration dir property before you call `parseAndValidate` method. I don't know why it's even working.

Reply:
Nice find, fixed. I will push some changes later today.